vcstools/gfit.py

```diff
@@ -62,80 +72,88 @@ class gfit:
         Weq_e: float
             The uncertainty in the equivalent width
         Wscat: float
-            The scattering width of the profile measured in phase
-        Wscat_e: float
-            The uncertainty in the scattering width
-        maxima: list
-            A lost of floats corresponding to the bin location of each maximum point
-        maxima_e: list
-            A list of floats, each correspinding to the error of the maxima of the same index. Measured in bins
-        redchisq: float
-            The reduced chi sqared of the fit
-        num_gauss: int
-            The number of gaussian components used in the best fit
-        bic: float
-            The Bayesian Information Criterion for the best fit
-        gaussian_params: list
-            A list of length 3*N there N is num_gauss. Each set of 3 parameters corresponds to the amp, centre and width of a guassian component
+            The scattering width of the profile measured in phaseprofile_region_from_pairs
+            The number of distinguishable profile components
```

Comment:
I may need to discuss this with you at some point if these make physical sense or how to get interesting science out of these numbers. For now the need units (and a few things above)